IrConsoleApp/Lecture6/Course.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace IrynaZharnasek.Lecture6
+{
+    public class Course
+    {
+        private string course;
+        private Mentor mentor;
+
+        private Student[] arrayOfStudents = new Student[10];
+        private Mentor[] arrayOfMentors = new Mentor[10];
+        private Course[] courses = new Course[10];
+
+        public string CourseTitle
+        {
+            get { return this.course; }
+        }
+
+        public Course(string course)
+        {
+            this.course = course;
+        }
+
+        public void AddNewStudent(Student student)
+        {
+            for (var i = 0; i < arrayOfStudents.Length; i++)
+            {
+                if (arrayOfStudents[i] == null)
+                {
+                    arrayOfStudents[i] = student;
+                    break;
+                }
+            }
+        }
+
+        public void AddNewMentor(Mentor mentor)
+        {
+            for (var i = 0; i < arrayOfMentors.Length; i++)
+            {
+                if (arrayOfMentors[i] == null)
+                {
+                    arrayOfMentors[i] = mentor;
+                    break;
+                }
+            }
+        }
```

Comment:
add empty line here
empty line should be before/after method

Reply:
Done